gitpod-io/gitpod
# Use veth instead of slirp4netns without affecting the supervisor

## Description

Use veth instead of slirp4netns without affecting the supervisor.

no2 of https://github.com/gitpod-io/gitpod/issues/8106#issuecomment-1094418016

## Related Issue(s)

Relates #8106

## How to test

on gitpod
```console
$ docker run hello-world
$ python -m http.server 5000 # and ensure to be able to access 5000 port from outward
```

## Release Notes

```release-note
Use veth instead of slirp4netns without affecting the supervisor
```

## Documentation
